ui/packages/consul-ui/app/components/consul/intention/notice/custom-resource/index.hbs

```diff
@@ -15,7 +15,14 @@ as |notice|>
   </notice.Body>
   <notice.Footer>
     <p>
-      <a href="{{env 'CONSUL_DOCS_URL'}}/k8s/crds" target="_blank" rel="noopener noreferrer">Learn more about CRDs</a>
+      <Hds::Button
+        @href="{{env 'CONSUL_DOCS_URL'}}/k8s/crds"
+        @text='Learn more about CRDs'
```

Comment:
Yes, it seems to be  that way according to the `Alert` docs in [Helios](https://helios.hashicorp.design/components/alert?tab=code).

Reply:
Updated here 
<img width="1369" alt="image" src="https://user-images.githubusercontent.com/10027860/219811567-f6c4c7bd-cb62-45af-8500-0d34cb3c9c98.png">

but for sure, it will be a part of `Alert` update to `Hds`